src/kendo.menu.js

```diff
@@ -462,6 +502,78 @@ var __meta__ = { // jshint ignore:line
             }
         },
 
+        _initOverflow: function(options) {
+            if (!options.overflow) {
+                return;
+            }
+
+            var that = this;
+            var isHorizontal = options.orientation == "horizontal";
+            var backwardBtn, forwardBtn;
+            that._openedPopups = {};
+            that._isRtl = kendo.support.isRtl(that.wrapper);
+            that._overflowWrapper = that.element.wrap("<div class='k-menu-scroll-wrapper " + options.orientation + "'></div>").parent();
+
+            if (isHorizontal) {
+                backwardBtn = $(templates.scrollButton({direction: "left"})).css({top: 0, left: 0, height: "100%"});
+                forwardBtn = $(templates.scrollButton({direction: "right"})).css({top: 0, right: 0, height: "100%"});
+            } else {
+                backwardBtn = $(templates.scrollButton({direction: "up"})).css({top: 0,left: 0, width: "100%"});
+                forwardBtn = $(templates.scrollButton({direction: "down"})).css({bottom: 0,left: 0, width: "100%"});
+            }
+            backwardBtn.appendTo(that._overflowWrapper);
+            forwardBtn.appendTo(that._overflowWrapper);

```

Comment:
backwardBtn and forwardBtn are always added and used together -we could try to use them as one template instead of two and just pass correct parameters.